app/src/main/java/com/keylesspalace/tusky/util/ShareShortcutHelper.kt

```diff
@@ -72,7 +71,7 @@ fun updateShortcut(context: Context, account: AccountEntity) {
         val intent = Intent(context, MainActivity::class.java).apply {
             action = Intent.ACTION_SEND
             type = "text/plain"
-            putExtra(NotificationHelper.ACCOUNT_ID, account.id)
+            putExtra(ShortcutManagerCompat.EXTRA_SHORTCUT_ID, account.id.toString())
         }
```

Comment:
Should this be moved in to a function in `MainActivity`, for consistency with all the others?

Reply:
Was wondering that as well, but it is very specific to the Shortcutmanager and should not be used anywhere else. Can still move if you think its better.